### ISSUE TITLE
Add metatag support for og:image:alt and twitter:image:alt.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/vets-website#15645]
This is the frontend companion of department-of-veterans-affairs/va.gov-cms#3563.  

Briefly, alt text is missing for images displayed in Facebook and Twitter cards.

The backend side is completed in [this PR](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/3965), which hasn't been merged yet.

I'll open a PR for the (quite small) changes shortly.

I didn't tag this as a `bug` because I'm not sure if this requirement is actually specified or implied anywhere and didn't want to overstep 😅
